lib/ansible/utils/path.py

```diff
@@ -107,6 +108,9 @@ def basedir(source):
 
 
 def cleanup_tmp_dir(path):
-    """Removes directory"""
+    """Removes temporary directory"""
     if os.path.isdir(path):
-        shutil.rmtree(path)
+        try:
+            shutil.rmtree(path)
+        except (FileNotFoundError, PermissionError):
+            print('Unable to remove temporary file', file=sys.stderr)
```

Comment:
no print, display

Reply:
I was going to ask about that. Do I just need to create a new display instance?

Reply:
Display is a singleton now, so just import and instantiate, then use.  See docs/docsite/rst/dev_guide/testing/sanity/no-main-display.rst

Reply:
I switched this to using `Display` imported within the function to avoid a circular import.